modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -61,7 +61,7 @@ spec:
 <3> Specify a name for the output.
 <4> Specify the `elasticsearch` type.
 <5> Specify the URL and port of the external Elasticsearch instance as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
-<6> If using an `https` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent.
+<6> If using an `https` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent. The secret must be of type `opaque` or `generic`.
```

Comment:
If using an `https` prefix and the external elasicsearch server is using private ca or you enabled mutual(client) authenticated to fluentd, you must specify the name of the secret required by the endpoint for TLS communication. To trust elasticsearch private ca, you must specify the key  ca-bundle.crt.  To enable mutual(client) authenticated, you must specified keys of: *tls.crt*, *tls.key.  The secret must exist in the `openshift-logging` project , the secret must be of type `opaque` or `generic`.

Reply:
@QiaolingTang  @igor-karpukhin  @jcantrill  Could you confirm?

Reply:
Im unable to comment if the type matters as I have never tried to different ones

Reply:
`To enable mutual(client) authenticated, you must specified keys of: tls.crt, *tls.key.` --> `To enable mutual(client) authenticated, you must specified keys of:**ca-bundle.crt**, tls.crt, tls.key.`

Reply:
@anpingli: with "mutual (client) authenticated" do you mean "mutual Transport Layer Security (mTLS)" ?

Reply:
Yes,mutual (client) authenticated = mutual Transport Layer Security (mTLS).

Reply:
@anpingli I've updated the sentence please confirm that its correct now. Thanks